src/docs/paths-opening-hours.yaml

```
@@ -0,0 +1,349 @@
+              properties:
+                day:
+                  type: string
+                  enum: [Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday]
+                  example: Monday
+                Working_hours:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      startTime:
+                        type: string 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)